x-pack/plugins/screenshotting/server/plugin.test.ts

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+jest.mock('./browsers/install');
+
+import type { ScreenshotModePluginSetup } from 'src/plugins/screenshot_mode/server';
+import { CoreSetup, CoreStart, PluginInitializerContext } from 'kibana/server';
+import { coreMock } from 'src/core/server/mocks';
+import { ScreenshottingPlugin } from './plugin';
+import { install } from './browsers/install';
+
+let initContext: PluginInitializerContext;
+let coreSetup: CoreSetup;
+let coreStart: CoreStart;
+let setupDeps: {
+  screenshotMode: ScreenshotModePluginSetup;
+};
+
+beforeEach(() => {
```

Comment:
Could you please wrap the test suite using describe blocks to have readable output?

```typescript
describe('ScreenshottingPlugin', () => {
  let initContext: PluginInitializerContext;
  let coreSetup: CoreSetup;
  let coreStart: CoreStart;
  let setupDeps: Parameters<ScreenshottingPlugin['setup']>[1];
  let plugin: ScreenshottingPlugin;

  beforeEach(() => {
    // ...
    plugin = new ScreenshottingPlugin(initContext);
  });

  describe('setup', () => {
    test('returns a setup contract', () => {
      // ...
    });

    test('handles setup issues', () => {
      // ...
    });
  });

  describe('start', () => {
    test('returns a start contract', () => {
      // ...
    });
  });
});
```

It will be easier to debug when we cover only one case (one expectation) in a single test.
The output, in this case, will look something like this:
```
ScreenshottingPlugin > setup > returns a setup contract
```